tests/test_install.py

```diff
@@ -125,8 +125,8 @@ def test_include_deps(pipx_temp_env, capsys):
 @pytest.mark.parametrize(
     "package_name, package_spec",
     [
-        ("zest-releaser", PKG["zest-releaser"]["spec"]),
         ("tox-ini-fmt", PKG["tox-ini-fmt"]["spec"]),
+        ("zest-releaser", PKG["zest-releaser"]["spec"]),
```

Comment:
Is this change significant? (I hope it’s not, otherwise this would be problematic)

Reply:
> Is this change significant? 

No, apologies, I was investigating a failing test. Reverted.

> (I hope it’s not, otherwise this would be problematic)

Out of curiosity, why that?